modules/nodes-containers-sysctls-unsafe.adoc

```diff
@@ -0,0 +1,110 @@
+// Module included in the following assemblies:
+//
+// * nodes/containers/nodes-containers-sysctls.adoc
+
+[id="nodes-containers-sysctls-unsafe_{context}"]
+= Enabling unsafe sysctls
+
+A cluster administrator can allow certain unsafe sysctls for very special
+situations such as high-performance or real-time application tuning.
+
+If you want to use unsafe sysctls, a cluster administrator must enable them
+individually for a specific type of node. The sysctls must be namespaced.
+
+[WARNING]
+====
+Due to their nature of being unsafe, the use of unsafe sysctls is
+at-your-own-risk and can lead to severe problems, such as improper behavior of
+containers, resource shortage, or breaking a node.
+====
+
+.Procedure
+
+. Add a label to the Machine Config Pool where the containers where containers 
+with the unsafe sysctls will run:
++
+[source,yaml]
+----
+$ oc edit machineconfigpool worker
+
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfigPool
+metadata:
+  creationTimestamp: 2019-02-08T14:52:39Z
+  generation: 1
+  labels:
+    custom-kubelet: sysctl <1>
+----
+<1> Add a `key: pair` label.
+
+. Create a KubeletConfig Custom Resource (CR):
++
+----
+apiVersion: machineconfiguration.openshift.io/v1
+kind: KubeletConfig
+metadata:
+  name: custom-kubelet
+spec:
+  machineConfigPoolSelector:
+    matchLabels:
+      custom-kubelet: sysctl <1>
+  kubeletConfig:
+    allowedUnsafeSysctls: <2>
+      - "kernel.msg*"
+      - "net.ipv4.route.min_pmtu"
+----
+<1> Specify the label from the Machine Config Pool.
+<2> List the unsafe sysctls you want to allow.
+ 
+. Create the object:
++
+----
+$ oc apply -f set-sysctl-worker.yaml
+----
++
+A new MachineConfig named in the `99-worker-XXXXXX-XXXXX-XXXX-XXXXX-kubelet` format is created.
+
+. Wait for the cluster to reboot usng the `machineconfigpool` object `status` fields:
++
+For example:
++
+----
+status:
+  conditions:
+    - lastTransitionTime: '2019-08-11T15:32:00Z'
+      message: >-
+        All nodes are updating to
+        rendered-worker-ccbfb5d2838d65013ab36300b7b3dc13
+      reason: ''
+      status: 'True'
+      type: Updating
+----
++
+A message similar to the following appears when the cluster is ready:
++
+----
+   - lastTransitionTime: '2019-08-11T16:00:00Z'
+      message: >-
+        All nodes are updated with
+        rendered-worker-ccbfb5d2838d65013ab36300b7b3dc13
+      reason: ''
+      status: 'True'
+      type: Updated
+----
+
+. When the cluster is ready, check for the merged `KubeletConfig` in the new MachineConfig:
++
+----
+oc get machineconfig 99-worker-XXXXXX-XXXXX-XXXX-XXXXX-kubelet -o json | grep ownerReference -A7
```

Comment:
`$` prompt at front, and maybe add a blank line after the command to give it some room from the output.